doc/particles.md

```diff
@@ -241,55 +238,58 @@ desired positioning.
 game.add(
   ParticleComponent(
     particle: CircleParticle(
-      radius: game.size.x / 2,
+      radius: game.size.width / 2,
```

Comment:
`game.size.x` was correct, or `game.width`

Reply:
Not sure how that got changed, fixed.